integreat_cms/cms/templates/analytics/_page_hix_widget.html

```diff
@@ -0,0 +1,46 @@
+{% extends "_collapsible_box.html" %}
+{% load i18n %}
+{% block collapsible_box_icon %}
+    gauge
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% translate "Overview of the text understandability of the pages" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    <div class="flex flex-wrap justify-between gap-2">
+        <p>
+            {% blocktranslate with n=page_translations|length %}
```

Comment:
```suggestion
            {% blocktranslate trimmed with n=page_translations|length %}
```
Ah sorry, my bad. could you add the "trimmed" parameter?